libs/database-entities/src/main/java/gov/cdc/nbs/entity/syphilis_qb/QuestionGroup.java

```diff
@@ -0,0 +1,36 @@
+package gov.cdc.nbs.entity.syphilis_qb;
+
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.OneToMany;
+import javax.persistence.FetchType;
+
+
+@Entity
+@Table(name = "question_group")
```

Comment:
I believe a `catalog` needs to be specified since there will be multiple schemas used